cmd/version.go

```diff
@@ -20,11 +26,45 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
-	},
+	Run: showVersion,
+}
+
+// showVersion outputs the version of the exercism binary that is in us.
+func showVersion(cmb *cobra.Command, args []string) {
+	cmdName := RootCmd.Use
+	fmt.Printf("%s version %s", cmdName, Version)
+
+	if checkLatest {
+		c := cli.New(Version)
+
+		ok, err := c.IsUpToDate()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if ok {
+			fmt.Println(", is up to date.")
+			return
+		}
+
+		cv, _ := semver.Make(c.Version)
+		rv, _ := semver.Make(c.LatestRelease.Version())
+
+		switch {
+		case cv.LT(rv):
+			fmt.Println(", is out of date.")
+			fmt.Printf("To upgrade to %s, see `%s upgrade`.\n", rv, cmdName)
+		case cv.GT(rv):
+			fmt.Printf(", is newer than the latest version %s\n", rv)
+		}
```

Comment:
I would avoid trying to DRY up the strings here. Let's make one message for "up to date" and one for "please upgrade". I'd much rather have some duplication and be able to read/edit the entire message at once.

We're eventually going to move these into configuration so that the API has control over them.